float32[] normal_toe
float32[] toes_force2 
float32[] toes_force1 
float32[] ground_friction 
float32[] height_scan
float32[] external_force
